test/model/message_test.dart

```diff
@@ -576,4 +603,197 @@ void main() {
         .reactions.isNotNull().jsonEquals([eg.unicodeEmojiReaction]);
     });
   });
+
+  group('handleSubmessageEvent', () {
```

Comment:
nit: order of tests should match order of code under test

(which in this case should, and does, match the order the event classes are defined)

Reply:
I think `handleDeleteMessageEvent` is out of order, will move that in a prep commit.